scripts/ansible/oe-windows-nonSGX-setup.yml

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+---
+- hosts: windows-agents
+  any_errors_fatal: true
+  become_method: runas
+  tasks:
+    - name: OE setup | Set installer URLs from the OE storage account
+      set_fact:
+        intel_psw_url:       "https://oejenkins.blob.core.windows.net/oejenkins/Intel%20SGX%20PSW%20for%20Windows%20v2.5.101.3.exe"
+        intel_psw_hash:      "D904964872A49426D3BFD0249752403DC746611F92F4FF6B95F64F336744323C"
+        intel_dcap_url:      "https://oejenkins.blob.core.windows.net/oejenkins/Intel%20SGX%20DCAP%20for%20Windows%20v1.3.101.3.exe"
+        intel_dcap_hash:     "27C1CC7F8434A704853FC3BFE15723F6F507CB09BF79B248A09A9ED0EED48A01"
```

Comment:
Heads up, on Friday we are going to need to change this due to Intel releasing their bits.

Reply:
I think this is out of scope for this PR , we can update that in a different PR when Intel releases the new version